src/pages/ExamsPage/components/AllowanceListActions.test.jsx

```diff
@@ -31,6 +50,13 @@ describe('AllowanceListActions', () => {
     expect(screen.getByText('Are you sure you want to delete this allowance?')).toBeInTheDocument();
   });
 
+  // todo: figure out why this isn't working
```

Comment:
Debugging this but the Edit allowance? is showing up as text in the render but the expect is failing, so must not be getting by text correctly. Will play around with this.